CDCgov/prime-reportstream
# Update precommit hook script to bypass klint if nothing has changed under /prime-router (#6842)

This PR updates `run-ktlintCheck.sh`, which is run as part of pre-commit checks by `git`, to skip running the ktlint checks in case `git status` reports no changes to the tree under `prime-router`.


Test Steps:
1. Execute `/bin/bash .git/hooks/pre-commit` from the repo's root folder with and without any changes to `prime-router` and observe the outcome. 

## Changes
- `run-ktlintCheck.sh` skips running the ktlint checks in case `git status` reports no changes to the tree under `prime-router`

## Checklist

### Testing

### Process
N/A

## Linked Issues
- Fixes #6842

## To Be Done
N/A

## Specific Security-related subjects a reviewer should pay specific attention to
N/A
